examples/experiments/ippo_ff_p1_self_play.yaml

```diff
@@ -49,7 +49,7 @@ train:
   resample_scenes: true
   resample_criterion: "global_step"
   resample_dataset_size: 500 # Number of unique scenes to sample from
-  resample_interval: 1_000_000
+  resample_interval: 20_000_000
```

Comment:
Small note: We probably want to swap the data batches more often because
1. We don't want the model to overfit to a single set of scenarios
2. We need to get through a lot of data throughout training